src/useToaster/useToaster.ts

```diff
@@ -13,7 +13,7 @@ import { ToastContainerProps } from '../toaster/ToastContainer';
  * @see https://rsuitejs.com/components/use-toaster/
  */
 const useToaster = () => {
-  const { toasters } = useContext(CustomContext);
+  const { toasters, toastContainer } = useContext(CustomContext);
```

Comment:
https://github.com/rsuite/rsuite/pull/4088/files#diff-a69afcc904e588ba7879d81366803d8fb43ed018e60a97c8719b83880ecd7c02R55
React Hook useMemo has a missing dependency: 'toastContainer'. Either include it or remove the dependency array